tests/wamr-test-suites/test_wamr.sh

```diff
@@ -39,7 +39,7 @@ function help()
     echo "-F set the firmware path used by qemu"
     echo "-C enable code coverage collect"
     echo "-j set the platform to test"
-    echo "-T set sanitizer to use in tests(ubsan|tsan|asan|posan)"
+    echo "-T set sanitizer(s) to use in tests (support comma-separated list, e.g. ubsan,asan) (ubsan|tsan|asan|posan)"
```

Comment:
```suggestion
    echo "-T set the sanitizer(s) used during testing. It can be either a comma-separated list (e.g., ubsan, asan) or a single option (e.g., ubsan, tsan, asan, posan)."
```